app/build.gradle

```diff
@@ -37,10 +37,16 @@ android {
     lintOptions {
         abortOnError false
     }
+
+    buildFeatures {
+        viewBinding = true
+    }
+
 }
 
 dependencies {
 
+    implementation 'androidx.databinding:library:3.2.0-alpha11'
```

Comment:
why do you need this lib? if it is needed please do not use alpha versions

Reply:
done